mockgcp/mock_http_roundtrip.go

```diff
@@ -64,7 +64,7 @@ import (
 	"github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/mockgcpregistry"
 	"github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/mockgkehub"
 	"github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/mockgkemulticloud"
-	"github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/mockiam"
+	_ "github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/mockiam"
```

Comment:
Mind adding a comment to explain this? I saw the storage is used in the mock IAM server